### DISCUSSION TITLE
"session not authenticated module=http" even tho one is logged in.

### DISCUSSION THREAD

**Discussion by fusk-l** [Budibase/budibase#13629] [Help]
Windows 11. Autobrr 1.41. Firefox 126.

From lan pc, suddenly being denied access. The webpage is just a black screen. When refreshing, for just a few ms, one can see the top bar on the dashboard and it is visible that it is logged in as the account username is visible in the upper right.
While continuing refreshing, you can see it is trying to load the dashboard, and also trying to post a notification about the error, but because it's only visible for ~200ms. You can't really make out what it says.

So, even tho you are logged in, the connection is denied by autobrr.

Since autobrr was installed it has worked fine. Looking through the logs, it appears it started randomly throwing the error "session not authenticated module=http" earlier today.

It's working on the host machine.

**Comment by Nephatiu** [comment]
I seemed to have this very same problem, updated (in hopes it would solve itself), and now it won't even let me in that microsecond anymore. On the page it now gives a 403 forbidden reply referring to "/api/config". Not sure how to get in without losing all my filters.

Did you find any solution to your problem that could help me with mine?
